luda-adventure/src/app/game/types/game_object/types/positioner/movement.rs

```diff
@@ -55,13 +55,13 @@ impl Movement {
             start_position: position,
             end_position: position,
             velocity: Xy::single(Per::new(0.tile(), 1.ms())),
-            movement_state: MovementState::Stuck,
+            collision_state: CollisionState::Stuck,
         }
     }
 }
 
 #[derive(Clone, Debug)]
-pub enum MovementState {
+pub enum CollisionState {
```

Comment:
So, it's value should be about collision, not movement.

Here is my suggestion.

- NotCollide
- WillCollide
- CollideOneAxis
- CollideOneAxisButWillBeTwo
- CollideTwoAxis

Reply:
No longer use this.